src/Contrib/StackSdks/Masa.Contrib.StackSdks.Auth/Service/UserService.cs

```diff
@@ -234,5 +234,23 @@ public async Task<bool> UpdatePhoneNumberAsync(UpdateUserPhoneNumberModel user)
         var requestUri = $"api/user/updatePhoneNumber";
         return await _caller.PutAsync<bool>(requestUri, user);
     }
+
+    public async Task<bool> LoginForPhoneNumberAsync(LoginForPhoneNumber login)
+    {
+        var requestUri = $"api/user/loginForPhoneNumber";
+        return await _caller.PostAsync<bool>(requestUri, login);
+    }
+
+    public async Task RemoveUserRolesAsync(RemoveUserRolesModel user)
+    {
+        var requestUri = $"api/user/removeUserRoles";
+        await _caller.DeleteAsync(requestUri, user);
+    }
+
+    public async Task<UserModel> AddThirdPartyUserAsync(AddThirdPartyUserModel user, bool whenExistReturn = true)
```

Comment:
https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=231&id=masastack_MASA.Framework&open=AYMVnTG2fHq-SUAraI4y

Reply:
no need to change

Reply:
@zhenlei520

Reply:
public async Task<UserModel> AddThirdPartyUserAsync(AddThirdPartyUserModel user, bool whenExistReturn = true)

-> public async Task<UserModel> AddThirdPartyUserAsync(AddThirdPartyUserModel user, bool whenExistReturn)

Reply:
changed